.azure/pipelines/devBuilds.yml

```diff
@@ -14,7 +14,7 @@ stages:
     parameters:
       codeSign: false
       jobName: Windows_build
-      jobDisplayName: "Build: Windows x64/x86"
+      jobDisplayName: "Build: Components"
```

Comment:
How long do each of these builds take? There is a cost to requiring more windows machines per pull request (finding more machines in the queue). It may be better to run build.cmd on the same machine, cleaning between runs as it would require one less windows machine.

Reply:
+1 on this, the x64/x86 job is already our long pole

Reply:
I actually plan to add two more jobs to this list - projectTemplates as well as building from the root. I'll talk to the @aspnet/build team folks tomorrow to see how I can optimize this further.

Reply:
Actually, thinking more about this, this pipeline is going to run once per day, so I think that this is not a concern. I can choose a safe time to run this.

Reply:
I've just added the schedule information to run every midnight. I think this is a relatively safe time to run this. @jkotalik, @wtgodbe does this resolve your concerns?

Reply:
Yep, that sounds good to me!